src/olympia/access/acl.py

```diff
@@ -151,10 +151,11 @@ def is_reviewer(request, addon):
     return check_addons_reviewer(request)
 
 
-def is_user_any_kind_of_reviewer(user):
+def is_user_any_kind_of_reviewer(user, allow_viewers=False):
```

Comment:
So how is this supposed to work? I don't see this argument referenced in the body of the function.

Reply:
Shouldn't we have new test cases for this change as well?

Reply:
Yeah I missed a bit. I'll implement that change and test for it.

Reply:
https://github.com/mozilla/addons-server/pull/13720/commits/a36a64bdac981e4da8a8c17ab043c49ce906aaac